<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>262</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>568</width>
    <height>262</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>568</width>
    <height>262</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>580</width>
    <height>281</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>580</width>
    <height>281</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Họ và tên</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mã sinh viên</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>190</y>
     <width>111</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
	border-radius: 2px;
	background-color: rgb(255, 230, 240);
	border: 1px solid #000;
}

QPushButton:hover{
	
	
	background-color: rgb(214, 205, 255);
}

QPushButton:focus {
	outline: none;
}

</string>
   </property>
   <property name="text">
    <string>Đăng nhập</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="classfield">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid #000;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="errorLogin">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lớp</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idfield">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid #000;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="namefield">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>12</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid #000;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::LogicalMoveStyle</enum>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>namefield</tabstop>
  <tabstop>idfield</tabstop>
  <tabstop>classfield</tabstop>
  <tabstop>loginButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
